ansible/roles/oso_console_extensions/tasks/main.yml

```diff
@@ -5,9 +5,42 @@
   when: item not in vars
   with_items: "{{ required_vars }}"
 
+# Install (docker-py) python package.
+- pip:
+    name: docker-py
+
 - import_tasks: uninstall.yml
   when: oso_ext_uninstall | bool
 
+- debug: msg="Building {{ oso_ext_builder_image}} builder image from {{ oso_ext_builder_repo }} ref {{ oso_ext_builder_git_ref }}"
+
+- name: 'Check for extistence of repo {{ oso_ext_builder_repo }}'
+  git:
+    repo: "{{ oso_ext_builder_repo }}"
+    version: "{{ oso_ext_builder_git_ref }}"
+    accept_hostkey: true
+    dest: /data/src/github.com/openshift/httpd-container
+    update: no
+  # Git may not be installed on remote hosts.
+  delegate_to: "{{ inventory_hostname }}"
```

Comment:
Did you mean `delegate_to: localhost`?

If Git is not installed on the remote host, how can you use it in the next step to fetch the Git repository?

Reply:
yea,  i removed that comment.  We can add 'Git' to prerequisites, i guess.  Also, delegate_to should be "{{ inventory_hostname }}" (this role should be run in OSEv3 host), bc the git clone happens in the OSEv3 host, not local.

Reply:
But `inventory_hostname` is the name of the remote host, which is where the task runs by default (if neither `delegate_to` nor `local_action` is specified), right? So if I understand correctly, the `delegates_to: "{{ inventory_hostname }}"` stanza is a no-op and can be omitted.